types/index.d.ts

```diff
@@ -0,0 +1,491 @@
+// Minimum TypeScript Version: 3.8
+declare module '@okta/okta-signin-widget';
+import { OktaAuth, OktaAuthOptions, Tokens } from '@okta/okta-auth-js';
+/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+export default class OktaSignIn implements OktaSignIn {
```

Comment:
Curious here, given this file is relatively big and is expected to grow bigger, should this be just generated from smaller files? It's going to get increasingly harder to test this file as well.